.repo-metadata.json

```diff
@@ -2,12 +2,13 @@
     "name": "sqlalchemy-bigquery",
     "name_pretty": "SQLAlchemy dialect for BigQuery",
     "client_documentation": "https://googleapis.dev/python/sqlalchemy-bigquery/latest/index.html",
-    "release_level": "beta",
+    "release_level": "preview",
     "language": "python",
     "library_type": "INTEGRATION",
     "repo": "googleapis/python-bigquery-sqlalchemy",
     "distribution_name": "sqlalchemy-bigquery",
     "api_id": "bigquery.googleapis.com",
     "default_version": "",
-    "codeowner_team": "@googleapis/api-bigquery"
+    "codeowner_team": "@googleapis/api-bigquery",
+    "api_shortname": "sqlalchemy-bigquery"
```

Comment:
This is not a GAPIC* library, is the `api_shortname` still required?

Reply:
Fixed